torch/testing/_internal/common_utils.py

```diff
@@ -99,6 +99,9 @@ def prof_meth_call(*args, **kwargs):
 parser.add_argument('--accept', action='store_true')
 parser.add_argument('--ge_config', type=str)
 parser.add_argument('--test_bailouts', action='store_true')
+parser.add_argument('--discover-tests', action='store_true')
```

Comment:
What's the purpose of this flag?

Reply:
It is similar to `--gtest-list-tests`, print list of tests to the console. 
Useful if one plans to implement external unit test runner (which builds database of tests and shards based on the average test runtime)